test/BUILD

```diff
@@ -1,6 +1,21 @@
 package(default_visibility = ["//visibility:public"])
 
-load("//bazel:envoy_build_system.bzl", "envoy_cc_test_library")
+load("//bazel:envoy_build_system.bzl", "envoy_cc_test", "envoy_cc_test_library")
+
+envoy_cc_test(
+    name = "example_configs_test",
+    srcs = [
+        "example_configs_test.cc",
+        "example_configs_test.h",
+    ],
+    data = ["//configs:example_configs"] + ["example_configs_test_setup.sh"],
```

Comment:
hmm. Very odd. I'm definitely running sandboxed on my machine. Potentially a difference in our host OS or bazel version? Will play with this more today.